bolt-aws-lambda/src/test/java/test_locally/Issue419UseCaseTest.java

```diff
@@ -79,6 +79,7 @@ public void enableAdditionalListenersRuntime_issue_419() throws Exception {
                 headers.put("additional-header", Arrays.asList("foo"));
                 return Response.builder().statusCode(200).headers(headers).build();
             });
+            app.message("Hello", (req, ctx) -> ctx.ack());
```

Comment:
Due to the current behavior, this test has been passing without this required handler.